### ISSUE TITLE
SMTP domain setting

### ISSUE THREAD

**Issue by lilianaziolek** [bitnami/bitnami-docker-discourse#112]
Hi, this is in relation to #56 and #75, that is, ability to set discourse's smtp_domain setting. I don't think it is extraneous - it seems to be required for email service providers such as sendgrid.
I am currently in process of migrating from Bitnami Discourse that is running on GCP to a dedicated VM - since the GCP launchers are not available, I decided to use the bundled docker image.

In my GCP config, I have the following settings:
```
smtp_address = "smtp.sendgrid.net"
smtp_port = 587
smtp_domain = XXX.com
smtp_user_name = 'apikey'
smtp_password = 'redacted'
smtp_enable_start_tls = true
authentication = login	 
```

This sends the emails which appear on clients as:
`XXX Discussion Forum <forum@XXX.com>`

When migrating to docker image run, the relevant section in docker-compose.yml is:
```
    environment:
      - SMTP_HOST="smtp.sendgrid.net"
      - SMTP_PORT=587
      - SMTP_USER='apikey'
      - SMTP_PASSWORD='redacted'
```
Unfortunately, sent emails appears as:
`XXX Discussion Forum forum@example.com via sendgrid.net`

This of course is not a great look, and I think it happens because the smtp_domain does not seem to be correctly sent to Sendgrid.
Is there any chance for you to expose SMTP_DOMAIN via docker env variables too?

As a temporary workaround, is it possible to expose the discourse/conf directory from the host system so that I could manually change the values in discourse.conf ? I tried doing it by mounting a volume but it seemed like it was creating some sort of infinite embedding of folders/symlinks.

**Comment by rafariossaa** [comment]
Hi, 
  I have created a internal task to add the support for configuring smtp_domain.

  It is not possible to mount only the conf directory. Meanwhile, as workaround you can modify it and then create your own image with this change.

**Comment by marcosbc** [comment]
Hi, FYI we just released a new container version, [2.7.0-debian-10-r4](https://github.com/bitnami/bitnami-docker-discourse#270-debian-10-r4). The initialization logic has received a major refactor, and its scripts are now located in the [`rootfs/` directory](https://github.com/bitnami/bitnami-docker-discourse/blob/master/2/debian-10/rootfs/).

It should be possible to add this new configuration via the following environment variable:

- `DISCOURSE_EXTRA_CONF_CONTENT`: Extra configuration to append to the `discourse.conf` configuration file. No defaults.
